### PR TITLE
Dextrose filtering

### PR DESCRIPTION
**Pull request by L0wry**
standardized filtering for lower and upper case components to be ignored

### CODE REVIEW

File: dextrose/dextrose.android.js [hunk 1]
```diff
@@ -1,23 +1,11 @@
 const path = require("path");
 
+const ignoredStories = require("./ignored-showcases");
```
Comment (craigbilner) [inline dextrose/dextrose.android.js]: showcases or stories :smirk:?

**Comment by times-tools** [comment]
If you use Expo, view our components by scanning this qr code: <br>  <br> This has been made possible through [Fructose](https://github.com/newsuk/fructose)

File: dextrose/dextrose.android.js (hunk 1 above)
Comment (L0wry) [reply dextrose/dextrose.android.js]: dextrose takes stories still should change this

Comment (L0wry) [reply dextrose/dextrose.android.js]: creating a pr now

**Comment by times-tools** [comment]
If you use Expo, view our components by scanning this qr code: <br>  <br> This has been made possible through [Fructose](https://github.com/newsuk/fructose)

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/f5239ce9b00e679dcbb99f9250e239edbe7b1553/index.html

**Comment by times-tools** [comment]
If you use Expo, view our components by scanning this qr code: <br>  <br> This has been made possible through [Fructose](https://github.com/newsuk/fructose)

**Pull request merged**